### PR TITLE
Adapt angular ngeo.editing

### PR DESCRIPTION
**Pull request by pfirpfel**

### CODE REVIEW

**Comment by pfirpfel** [comment]
todo: 
src/directives/exportfeatures.js (extra)

File: src/editing/createfeaturecomponent.js [hunk 1]
```diff
@@ -15,6 +16,12 @@ goog.require('ol.style.Style');
 // Don't forget to add module dependencies
```
Comment (ger-benjamin) [inline src/editing/createfeaturecomponent.js]: remove this line please

**Comment by ger-benjamin** [comment]
You do not have a extraModule.js ?

**Comment by pfirpfel** [comment]
I will add this asap.

**Comment by pfirpfel** [comment]
Found a typo (added here as a todo):

https://github.com/camptocamp/ngeo/blob/bf08afee1f0c7fac05cac2098dc09fdfc628f2c2/src/misc/extraModule.js#L20

**Comment by ger-benjamin** [comment]
Nice catch, thanks !

**Pull request merged**